src/components/qas/questions/qaCalculator/equations/FindMass.jsx

```diff
@@ -1,75 +1,39 @@
+/* eslint-disable react/no-danger */
+/* eslint-disable no-underscore-dangle */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import { QACalculatorIconUnit } from '../qQaCalculatorIcons.jsx';
-import { addTheCommas, toSigFigs } from '../../../../../lib/utilities.js';
+import MathJax from '../../../../site/mathJax';
 import {
-  color,
-  fraction,
-  denominator,
-  equation,
-  exponent,
-} from '../qaCalculator.module.scss';
-import { qaReviewHighlight } from './equations.module.scss';
+  addTheCommas,
+  scientificNotation,
+} from '../../../../../lib/utilities.js';
+import { renderUnit } from '../../../../charts/shared/unit/utilities.js';
+import { primary } from '../../../../../assets/stylesheets/_variables.scss';
+
+function addColorAndUnit(value, unit) {
+  return `\\textcolor{${primary}}{${value}} \\ \\mathrm{${renderUnit(unit)}}`;
+}
```

Comment:
I would expect this is gonna be something you'll use in every calculator, so let's abstract this to a utilities file that's in the same directory as your MathJax component and then you can either import the color in that file, or you can add color as an argument to `addColorAndUnit` and pass it in that way.